app/load.php

```diff
@@ -2,4 +2,4 @@
 
 require __DIR__.'/../vendor.phar';
 
-return \Dxw\Iguana\Init::init(__DIR__, GovUKBlogs::class);
+return \Dxw\Iguana\Init::init(__DIR__, \GovUKBlogs::class);
```

Comment:
@mallorydxw I thought the convention was for this to be the namespace as a string (e.g. `return \Dxw\Iguana\Init::init(__DIR__, 'GovUKBlogs');`), rather than a class reference?

Reply:
I think using `::class` seems a little cleaner. If there are backslashes then they don't need to be escaped. And they're identical:

```
% php -r 'echo \Xyz::class === "Xyz";'
1
```